app/controllers/api/v1/enlistment_controller.rb

```diff
@@ -54,6 +54,7 @@ def find_project
   end
 
   def delete_all_subscriptions(code_location_id)
-    Enlistment.connection.execute("DELETE FROM fis.subscriptions WHERE code_location_id =#{code_location_id};")
+    Enlistment.connection.execute("DELETE FROM fis.subscriptions WHERE code_location_id =
+                                  #{ActiveRecord::Base.sanitize_sql(code_location_id)};")
```

Comment:
This `code_location_id` comes directly from the DB. We shouldn't need to sanitize it here. You may undo this change.
Is there any way to mark this line as safe for coverity scan?